bin/run-reference-solution-tests-in-docker.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Synopsis:
+# Test the test runner Docker image by running it against all reference solutions from the Groovy repository.
+# It is recommended to run this script when upgrading Groovy exercises or Groovy test runner dependencies.
+
+# Output:
+# Fails is
+
+# Example:
+# ./bin/run-reference-solution-tests-in-docker.sh
+
+# Build the Docker image
+docker build --rm -t exercism/groovy-test-runner .
+export BUILD_IMAGE=false
+
+# Assumes exercism/groovy repository is checked out next to the exercism/groovy-test-runner repository.
+# If the repository is checked out elsewhere, provide the path as the first argument.
+groovy_repo_path=$(realpath "${1:-${PWD}/../groovy}")
+repo_root=$(git rev-parse --show-toplevel)
+rm -r "${repo_root}/tmp" || true
+
+# Iterate over all exercises directories
+for exercise_dir in "${groovy_repo_path}"/exercises/practice/*; do
+  exercise_slug=$(basename "${exercise_dir}")
```

Comment:
Indentation is usually 4 spaces. 
You don't need to run external commands. You can do `slug=${exercise_dir##*/}`

Reply:
Fixed